src/test/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/wiremock/MappingMockServer.kt

```diff
@@ -110,6 +110,71 @@ class MappingMockServer : WireMockServer(WIREMOCK_PORT) {
     )
   }
```

Comment:
unfortunately i couldnt split this into 2 classes as you cant have 2 Mockservers on the same port